web/.eslintrc.json

```diff
@@ -15,6 +15,7 @@
     "no-param-reassign": 0,
     "import/prefer-default-export": 0,
     "prefer-template": 0,
+    "react/destructuring-assignment": 0,
```

Comment:
forced destructuring has single handedly improved my teams codebase at work by 100%

Reply:
because I can't do stuff like https://github.com/nwhacks/nwhacks2019/pull/62/files#diff-1c485da269908c430e2793fab9c4577aR55 (like there's literally no workaround that i know of)

if you know of any workaround, i will revert this :smile:

Reply:
oohhhh it just wants you to deconstruct `props`, not literally everything. okay, will revert.

and yeah, that's pretty cool!!